### PR TITLE
PR for #67: Implement new main table and figure

### PR DESCRIPTION
**Pull request by santiagohermo**
Can you take a look at this PR for #67, @gabrieleborg? Change here:

- I implemented the new main figure with two panels we talked about before. Did not implement a "long-differences" estimate since it's not clear how to go about it.
- I updated the main table with economic controls only. I also changed the number of decimals to 4.
- I changed the structure of the `first_differences` folder and updated a bit the code. I sent a few functions to [the `lib` folder as ado files](https://github.com/diegogentilepassaro/min_wage_rent/tree/0bd62f3ceec62e1953f8cc3b27964e330a960c89/lib/stata/min_wage/ado). 

An important note (fyi @diegogentilepassaro): as you can see in [this code](https://github.com/diegogentilepassaro/min_wage_rent/blob/0bd62f3ceec62e1953f8cc3b27964e330a960c89/lib/stata/min_wage/ado/define_controls.ado), we use as controls the sectors "Professional and Business Services", "Information", and "Manufacturing". A full list of available sectors in the QCEW is [here](https://www.bls.gov/cew/classifications/industry/industry-supersectors.htm). Let me know if you think we should use other sectors and we can update the folder.

You can see the resulting figures and tables in [staged output](https://github.com/diegogentilepassaro/min_wage_rent/blob/0bd62f3ceec62e1953f8cc3b27964e330a960c89/paper_slides/staged_output/output.pdf).

### CODE REVIEW

**Comment by santiagohermo** [comment]
I will implement updates in the draft following https://github.com/diegogentilepassaro/min_wage_rent/issues/67#issuecomment-721210964 in #65 @gabrieleborg

**Review by gabrieleborg — CHANGES_REQUESTED** [review]
Thanks for the great update @santiagohermo ! It all looks ok to me. I just left a comment/remainder regarding the model specification we use for the plots. I was wondering whether you could try to switch the manufacturing sector with the finance one...does the result change significantly? because if not, I feel there is a stronger case for saying that that is an unlikely sector to be affected by MW policies

**Comment by gabrieleborg** [comment]
@diegogentilepassaro @santiagohermo I was thinking I could use this PR to update the other parts of the analysis too? As per our conversation last week, I wanted to: 

- update the various weighted, unbalanced panel specification now reflecting the structure Santi implemented in the main first_difference folder
- update the baseline model including Economic controls (let me know if we are on the same page on this, but I believe we said that since we are introducing them since the beginning this should become our baseline specification)

If you want me to open a different issue, just let me know 👍🏽

File: analysis/first_differences/code/figures.do [hunk 1]
```diff
@@ -11,20 +11,28 @@ program main
...
-	graph export "`outstub'/fd_models.eps", replace
+		cluster(statefips) outstub(`outstub')
+
+	foreach w in 3 6 9 {
+		use "`instub'/fd_rent_panel.dta", clear
+
+		build_coeff_plot, depvar(ln_med_rent_psqft_sfcc) absorb(year_month) ///
+			cluster(statefips) outstub(`outstub') w(`w')
+	}
+
```
[the remark this answers is not in the document]
Comment (santiagohermo) [reply analysis/first_differences/code/figures.do:22]: Yes! I wasn't sure whether we want to include them or not in the figures. In any case, cased on the dynamic table I expect the coefficients to be very similar. Should I include them @gabrieleborg @diegogentilepassaro?

File: analysis/first_differences/code/figures.do [hunk 2]
```diff
@@ -11,20 +11,28 @@ program main
...
+	use "`instub'/fd_rent_panel.dta", clear
+	
+	build_cumsum_plot, depvar(ln_med_rent_psqft_sfcc) absorb(year_month) ///
+		cluster(statefips) outstub(`outstub')
```
[the remark this answers is not in the document]
Comment (santiagohermo) [reply analysis/first_differences/code/figures.do:26]: Yes!

**Comment by santiagohermo** [comment]
Thanks @gabrieleborg! Regarding your comment about using the financial sector. Unfortunately, we were not including it in `base/qcew`  (see [here](https://github.com/diegogentilepassaro/min_wage_rent/blob/0bd62f3ceec62e1953f8cc3b27964e330a960c89/base/qcew/code/make_county_month_panel.do#L66-L78)). Now I'm running that step again, and I will use it instead of the manufacturing sector. As soon as it's done I'll update results and merge here (we also need to decide whether to use controls or not in the regressions, but that decision can be made later with no problem)

**Pull request merged**